pandas/tests/frame/test_dtypes.py

```diff
@@ -762,6 +762,12 @@ def test_astype_categoricaldtype_class_raises(self, cls):
         with pytest.raises(TypeError, match=xpr):
             df["A"].astype(cls)
 
+    def test_singlerow_slice_categoricaldtype_gives_series(self):
+        df = pd.DataFrame({"x": pd.Categorical("a b c d e".split())})
+        raw_cat = pd.Categorical(["a"], categories=["a", "b", "c", "d", "e"])
+        expected = pd.Series(raw_cat, index=["x"], name=0, dtype="category")
+        tm.assert_series_equal(df.iloc[0], expected)
```

Comment:
Can you use `tm.assert_series_equal(result, expected)`

Reply:
To clarify @mroeschke comment, he means this:

~~~python
result = df.iloc[0]
tm.assert_series_equal(df.iloc[0], expected)
~~~

And while we're at it, let's also add a new line beneath `expected` to make it easier to read.

Reply:
Done